README.md

```diff
@@ -1,21 +1,235 @@
 
 </div>
 
-JFrog IntelliJ IDEA plugin adds JFrog Xray scanning of Maven, Gradle, Go, Npm, and PyPI project dependencies to your IntelliJ IDEA.
-To learn how to use JFrog IntelliJ IDEA plugin, please visit the [JFrog IntelliJ IDEA Plugin User Guide](https://www.jfrog.com/confluence/display/XRAY/IDE+Integration).
+# Table of Contents
+- [JFrog IntelliJ IDEA Plugin](#jfrog-intellij-idea-plugin)
+- [Table of Contents](#table-of-contents)
+- [About this Plugin](#about-this-plugin)
+- [Getting Started](#getting-started)
+  - [Set Up a FREE JFrog Environment in the Cloud](#set-up-a-free-jfrog-environment-in-the-cloud)
+  - [Connecting the Plugin to Your JFrog Environment](#connecting-the-plugin-to-your-jfrog-environment)
+    - [In the IDE Settings](#in-the-ide-settings)
+    - [Using Environment Variables](#using-environment-variables)
+  - [Applying Your Xray Policies](#applying-your-xray-policies)
+- [Using the Plugin](#using-the-plugin)
+  - [The Local View](#the-local-view)
+      - [Scanning a Project](#scanning-a-project)
+      - [Viewing Vulnerability Details](#viewing-vulnerability-details)
+      - [Contextual Analysis](#contextual-analysis)
+      - [Dependencies Tree Icons](#dependencies-tree-icons)
+  - [The CI View](#the-ci-view)
+      - [How Does It Work?](#how-does-it-work)
+      - [Setting Up CI Integration](#setting-up-ci-integration)
+- [Troubleshooting](#troubleshooting)
+- [Reporting Issues](#reporting-issues)
+- [Contributions](#contributions)
+  - [Building and Testing the Sources](#building-and-testing-the-sources)
+  - [Developing the Plugin Code](#developing-the-plugin-code)
+  - [Code Contributions](#code-contributions)
+- [Release Notes](#release-notes)
 
-# Building and Testing the Sources
+# About this Plugin
+The cost of remediating a vulnerability is akin to the cost of fixing a bug.
+The earlier you remediate a vulnerability in the release cycle, the lower the cost.
+[JFrog Xray](https://jfrog.com/xray/) is instrumental in flagging components when vulnerabilities are discovered in production systems at runtime,
+or even sooner, during the development.
+
+JFrog IntelliJ IDEA Plugin adds JFrog Xray scanning of project dependencies to IntelliJ IDEA.
+It allows developers to view panels displaying vulnerability information about the components and their dependencies directly in their IDE.
+The plugin also allows developers to track the status of the code while it is being built, tested and scanned on the CI server.
+
+Currently, Maven, Gradle, npm, Yarn, Python and Go are supported by the plugin.
+
+In addition to IntelliJ IDEA, the plugin also supports the following IDEs:
+* WebStorm
+* PyCharm
+* Android Studio
+* GoLand
+
+# Getting Started
+1. Install the JFrog IntelliJ IDEA Plugin via the Plugins tab in the IDE settings, or in [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/9834-jfrog).
```

Comment:
Let's add an image here, showing how the plugin can be installed from within IDEA.

Reply:
It'll be in a new paragraph.